src/ui/src/favoritefiles.cpp

```diff
@@ -80,7 +80,8 @@ void FavoriteFiles::retrieveFromStorage( QSettings& settings )
     if ( settings.contains( "FavoriteFiles/version" ) ) {
         settings.beginGroup( "FavoriteFiles" );
         if ( settings.value( "version" ).toInt() == FavoriteFilesVersion ) {
-            int size = settings.beginReadArray( "favorites" );
+            const int size = settings.beginReadArray( "favorites" );
```

Comment:
Another reading settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3